Lib/tkinter/ttk.py

```diff
@@ -1151,6 +1151,24 @@ def __init__(self, master=None, **kw):
         Widget.__init__(self, master, "ttk::sizegrip", kw)
 
 
+class Spinbox(Entry):
+    """Ttk Spinbox is an Entry with increment and decrement arrows, used
+     for number entry"""
```

Comment:
Ah yes, I'd forgotten it can take a list of values, just like a combobox.  I think the relevant methods are already in `ttk.combobox`, would it make sense to inherit from that class or would it be better to reimplement them here?  It's not much code, but I don't think there's anything unique to add.

Reply:
Given that the tk docs say that ttk spinbox *is* a ttk entry with added arrow buttons, I would think that Entry rather than Combobox is the correct baseclass.